### PR TITLE
Fix typo in README

### PR DESCRIPTION
**Pull request by stp-ip**

### CODE REVIEW

**Comment by ojarjur** [comment]
Hi, thanks for the pull request. Before we merge it, can you please go to https://cla.developers.google.com/ and sign the Google contributor license agreement?

**Comment by stp-ip** [comment]
Done. Thought cla was only needed for 1+ line changes so.

File: README.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ clear that these are meant to be read and written by automated tools.
 
 When a field named "v" appears in one of these notes, it is used to denote
 the version of the metadata format being used. If that field is missing, then
-it defaults to the value 0, which corresponds to this initial verison of the
+it defaults to the value 0, which corresponds to this initial version of the
 formats.
 
 ### Code Review Requests
```
Comment (lealon2014) [inline README.md]: 283422

**Comment by lealon2014** [comment]
That's that

**Pull request merged**